SEED-platform/seed
# allow auto update via audit template tool

#### Any background context you want to provide?
Users will need to automatically update a property using Audit Template XML.

#### What's this PR do?
- adds Audit Template Organization ID, Audit Template Email and Audit Template Password to the organization settings.
- adds endpoints to generate an Audit Template API key and to pull a building's XML
- adds actions to the property detail page to update via Audit Template

#### How should this be manually tested?
- You will need your Audit Template Organization ID, Audit Template Email and Audit Template Password saved on your organization.
- navigate to a property's detail page
- use the action to upload an XML and test property updates
- use the action to update via Audit Template and enter the AT building ID and verify property updates

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/3219

#### Screenshots (if appropriate)
<img width="619" alt="Screen Shot 2022-07-11 at 4 09 21 PM" src="https://user-images.githubusercontent.com/564231/178367622-5e992999-993c-4f82-b41f-70332da86fed.png">
